src/RemoteSigner.js

```diff
@@ -80,23 +80,31 @@ class RemoteSigner extends Ethers.Signer {
 
   /**
    * Sign a hashed piece of data
-   * @param {String} digest - Hex string of hashed data
+   * @param {String} digest - string of hashed data
+   * @param {Boolean} usePersonal - use EIP-191 personal_sign
    * @returns - the signed message as a hex string
    */
-  async signDigest(digest) {
+  async signDigest(digest, usePersonal=false) {
     if(!this.signatureCache[digest]) {
       this.signatureCache[digest] = new Promise(async (resolve, reject) => {
         try {
+          let path;
+          let body;
+          if(usePersonal) {
+            path = UrlJoin("as", "wlt", "sign", "personal", this.id);
+            body = { message: digest };
+          } else {
+            path = UrlJoin("as", "wlt", "sign", "eth", this.id);
+            body = { hash: digest };
+          }
           let signature = await Utils.ResponseToJson(
             this.HttpClient.Request({
               method: "POST",
-              path: UrlJoin("as", "wlt", "sign", "eth", this.id),
+              path: path,
```

Comment:
if variable name is the same as key name, you can just have it once, e.g. `{ method: "POST", path, ... }`

Reply:
re. my other comment above, I think instead of making this a parameter of signDigest, it should actually be implemented as signMessage instead